curvefs/src/mds/fs_manager.cpp

```diff
@@ -1280,5 +1289,10 @@ bool FsManager::FillVolumeInfo(common::Volume* volume) {
     return true;
 }
 
+void FsManager::SetClientMdsAddrsOverride(const std::string& addrs) {
+    WriteLockGuard lock(clientMdsAddrsOverrideMutex_);
+    clientMdsAddrsOverride_ = addrs;
```

Comment:
It's better to do some basic verification of the IP addresses, otherwise, it may cause client exception.

Another point is that it is better to update the addresses gradually, you cannot change all addresses directly. For example, you must change from (A,B,C) -> (A,B,C,D,E,F) -> (D, E, F).

Reply:
BTW, should you persist mds addresses into etcd? Becuase there's chance that mds exit before client get new addresses.

Reply:
> It's better to do some basic verification of the IP addresses, otherwise, it may cause client exception.
> 
> Another point is that it is better to update the addresses gradually, you cannot change all addresses directly. For example, you must change from (A,B,C) -> (A,B,C,D,E,F) -> (D, E, F).

ok i will make sure active mds is in the override ones

Reply:
> BTW, should you persist mds addresses into etcd? Becuase there's chance that mds exit before client get new addresses.

no. it's just a temporary online advanced setting.